cli/src/main/java/com/devonfw/tools/ide/context/AbstractIdeContext.java

```diff
@@ -819,9 +803,7 @@ private ValidationResult applyAndRun(CliArguments arguments, Commandlet cmd) {
     }
     if (result.isValid()) {
       debug("Running commandlet {}", cmd);
-      if (cmd.isIdeHomeRequired() && (this.ideHome == null)) {
-        throw new CliException(getMessageIdeHomeNotFound(), ProcessResult.NO_IDE_HOME);
-      } else if (cmd.isIdeRootRequired() && (this.ideRoot == null)) {
```

Comment:
You should not remove this.
It will cause strange errors otherwise and therefore this handling must stay.

Reply:
Thanks, I readded it